packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -28,6 +28,28 @@ export enum AppType {
   Web = "web",
 }
 
+/**
+ * Parse the input text and get the associated AppType
+ * @param text app-type/platform text
+ * @returns AppType or undefined.
+ */
+export function parseAppType(text: string | undefined): AppType | undefined {
+  switch (text ? text.toLowerCase() : undefined) {
+    case "desktop":
+    case "macos":
```

Comment:
this should only have "desktop" and "web" and not the other platforms (but keep undefined case). For the record there is a separate function for parsing the platforms and you can map from platform to app type. office-addin-debugging start does this, but it's not needed for sideload.

Reply:
Wait I am confused - do we not need all the stuff anymore about macOs, win32, ios and android? Are these effectively no longer used anywhere. If I remove them from here, will that cause problems for the office-addin-debugging call to parseAppType in packages/office-addin-debugging/src/commands.ts?

Reply:
If you look at this diff, the code you're moving doesn't have those other cases. Those cases are part of parsePlatform() in office-addin-debugging. We don't need all of the platforms here and we could add support for it later if for some reason we did, e.g. sideload to android.

Reply:
actually I think I am mistaken.

Reply:
This is the code I was thinking of: https://github.com/OfficeDev/Office-Addin-Scripts/blob/755485666db67538ed802277efad751f1c01565e/packages/office-addin-debugging/src/commands.ts#L43

However, you're right that this function did have the platform strings. Sorry for the confusion.